### ISSUE TITLE
iCloud3 v3.1.3 ➤ A new release of iCloud3 v3 is available - 11/9/2024

### ISSUE THREAD

**Issue by gcobb321** [gcobb321/icloud3#394]
### Change Log - v3.1.3 - 11/9/2024
1. UPDATE DEVICES SCREEN (Fixed) - Fixed a problem where an orphaned Mobile App device would prevent the Update Screen loading.

**Comment by cedricdelecole** [comment]
Hello, the documentation has not been updated yet with the version 1.3.1 and the introduction of multiple apple accounts (iso using family) ?
Many Thanks !!

**Comment by gcobb321** [comment]
No, the documentation has not been updated yet. I have been extremely busy just getting the code working and released after Apple changed their password verification process. v3.1 and its updates, v3.1.x, were rushed out before it. The basic screen formats and operation are described in the docs, just not the new stuff.

**Comment by cedricdelecole** [comment]
Thanks a lot.for that !!

**Comment by cedricdelecole** [comment]
Also assume I have basic question (unclear from the documentation), like on position update interval, where can I ask this (this is not an issue, just a question ...) ?
There is no "dedicated forum" on IC V3 I believe ?

Many thanks !!
